libs/secure-sm/src/test/java/org/elasticsearch/secure_sm/SecureSMTests.java

```diff
@@ -20,8 +24,13 @@
 import java.util.stream.Collectors;
 
 /** Simple tests for SecureSM */
-public class SecureSMTests extends TestCase {
-    static {
+public class SecureSMTests {
+
+    @BeforeClass
+    public static void initializeSM() {
+        if (RuntimeVersionFeature.isSecurityManagerAvailable() == false) {
```

Comment:
this could just be `assumeFalse(RuntimeVersionFeature.isSecurityManagerAvailable())`?

Reply:
Yes 🤦

Reply:
Oh wait, no, SecureSMTests is a pure JUnit test, assumeFalse is LuceneTestCase - I knew there was a reason :)

Reply:
Assume false in Lucene delegates to randomized testing. Use `RandomizedTest.assumeFalse`

Reply:
Good to know, changed it